### DISCUSSION TITLE
Wiki Question

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#419] [Q&A]
Hallo @spcqike

ich wollte meine Seite 
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter-%E2%80%90-Solar-Passthrough
**unter** deiner Seite 
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter
platzieren.

Weiss jemand wie ich das erreichen kann?

**Comment by spcqike** [comment]
Meinst du, wie es bei "Home" auch ist?

Abschnitte können mit "#" oder "##" gemacht werden

> 
> # neue große Überschrift  (Zeile beginnt mit #)
>   text
> 
> ## neue kleine Überschrift (Zeile beginnt mit ##)
>    text

**Reply by Manos1966** [reply]
Danke

**Reply by spcqike** [reply]
kleine Anmerkung
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/50098985/16b60bc9-c546-4ffc-8381-3d57174e4931)

Ein Neustart braucht es nur bei einer Batterie. ohne geht der WR nachts so oder so aus :) 
im Gegenteil: du kannst um Mitternacht keinen Neustartbefehl senden/ausführen, da der WR nicht an ist.

**Reply by Manos1966** [reply]
oh, stimmt! korrigiere ich.
Nächstes Mal kannst du direkt die Änderungen/ Ergänzungen auf der Seite machen ;-)

**Reply by spcqike** [reply]
ich war mir nicht sicher, wie du die Seiten nun genau aufbauen willst :) ob einzeln (wie ich deine Frage verstanden hab) oder getrennt (wie sie aktuell sind) 😄 da wollt ich nicht dazwischen hauen ;) 

beim nächsten mal dann 😃

**Reply by Manos1966** [reply]
Bin ich auch genauso wenig sicher...
Da wir aber kaum etwas in der Wiki drin haben, dachte ich es ist besser erst ein paar Sachen rein zu tun (Deutsch/Englisch).
Und da jeder Schreib-Zugriff auf die Wiki hat, dann kann jeder alle Seiten (nach besten Wissen und Gewissen) anpassen.

Ich persoenlich mag separate Seiten, weil man kann Sachen einfacher korrigieren/ergaenzen. Geschmackssache eben....
So habe ich meine Seiten in deiner addiert:

![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/521109fa-a044-41a9-a768-9001bc89ec38)
